### PR TITLE
refine install.md: make commands more clean.

### PR DESCRIPTION
**Pull request by yokofly**
127.1: too verbose and it shall work for now.
-C/etc/config.yaml:  this is only the most easy to edit config in binary, but it's hard to upgrade and maintain.
brew: https://docs.brew.sh/Taps#formula-with-duplicate-names:~:text=If%20you%20need,behaviour%20would%20be%3A

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -21,14 +21,13 @@ curl https://install.timeplus.com/oss | sh
...
 
 - **Local Mode.** You run `proton local` to start it for fast processing on local and remote files using SQL without having to install a full server
-- **Config-less Mode.** You run `proton server` to start the server and put the config/logs/data in the current folder. Then use `proton client -h 127.0.0.1` in the other terminal to start the SQL client.
-- **Server Mode.** You run `sudo proton install` to install the server in predefined path and a default configuration file. Then you can run `sudo proton server -C/etc/proton-server/config.yaml` to start the server and use `proton client` in the other terminal to start the SQL client.
+- **Config-less Mode.** You run `proton server` to start the server and put the config/logs/data in the current folder `proton-data`. Then use `proton client` in the other terminal to start the SQL client.
+- **Server Mode.** You run `sudo proton install` to install the server in predefined path and a default configuration file. Then you can run `sudo proton server -C /etc/proton-server/config.yaml` to start the server and use `proton client` in the other terminal to start the SQL client.
```
Comment (jovezhong) [inline docs/install.md:25]: I do noticed -C/path/to/yaml file works, even it's a bit surprising. Can you share me the code for how this -C is handled?

File: docs/install.md [hunk 2]
```diff
@@ -21,14 +21,13 @@ curl https://install.timeplus.com/oss | sh
...
 
 For Mac users, you can also use [Homebrew](https://brew.sh/) to manage the install/upgrade/uninstall:
 
 ```shell
-brew tap timeplus-io/timeplus
-brew install proton
+brew install timeplus-io/timeplus/proton
```
Comment (jovezhong) [inline docs/install.md:30]: The difference is users only need to run `brew tap timeplus-io/timeplus` once and later on can use `brew install` or ` brew upgrade` to install new versions

Comment (yokofly) [reply docs/install.md:30]: ok, understand, then this way is helpful.
but I guess fewer people will upgrade.
do u think: we shall send a telemetry on how many user will upgrade?

File: docs/install.md (hunk 1 above)
Comment (yokofly) [reply docs/install.md:25]: also surprised me. yes, I already tried, it works.
but I believe this is not straightforward, for me, I probably first copy paste and then manually type a blank.

below code is done by bash script, but I compared with clickhouse latest one, seems they also has this `feature???`
```
➜  ck ./clickhouse server -C../proton-enterprise/programs/server/config.yaml
Processing configuration file '../proton-enterprise/programs/server/config.yaml'.
Merging configuration file '../proton-enterprise/programs/server/config.d/access_control.xml'.
Poco::Exception. Code: 1000, e.code() = 0, Exception: Failed to merge config with '../proton-enterprise/programs/server/config.d/access_control.xml': Exception: Root element doesn't have the corresponding root element as the config file. It must be <clickhouse>, Stack trace (when copying this message, always include the lines below):
0. DB::ConfigProcessor::processConfig(bool*, zkutil::ZooKeeperNodeCache*, std::shared_ptr<Poco::Event> const&) @ 0x0000000013513b2a
1. DB::ConfigProcessor::loadConfig(bool) @ 0x0000000013513e54
2. BaseDaemon::initialize(Poco::Util::Application&) @ 0x000000000da7be40
3. DB::Server::initialize(Poco::Util::Application&) @ 0x000000000d955898
4. Poco::Util::Application::run() @ 0x0000000015bd187a
5. DB::Server::run() @ 0x000000000d955650
6. Poco::Util::ServerApplication::run(int, char**) @ 0x0000000015bda6c7
7. mainEntryClickHouseServer(int, char**) @ 0x000000000d9528a6
8. main @ 0x0000000007b7000e
9. ? @ 0x00007fecd1cc4083
10. _start @ 0x000000000625302e
 (version 24.8.1.1666 (official build))
➜  ck
```


https://github.com/timeplus-io/proton/blob/d5dc45fa9ac91e65a949b4041eab8ade2b1e17d5/programs/bash-completion/completions/proton-bootstrap#L150-L165

File: docs/install.md (hunk 2 above)
Comment (jovezhong) [reply docs/install.md:30]: I am not sure how homebrew sends the upgrade telemetry. For proton, if they has some local data then new version tries to use the older data, we may have a chance to send such upgrade event. But if they use proton in local mode or as a client, we may not know. Anyway, let me merge the proposed changes. Thanks for the PR

**Review by jovezhong — APPROVED** [review]

**Pull request merged**